_layouts/product-get-started.html.slim

```diff
@@ -10,14 +10,14 @@ h2#get-started Get Started
 
 h3.divider#installation-instructions Instructions for Installation
 
-= partial "os-selector.html.slim"  
+= partial "os-selector.html.slim"
```

Comment:
Don't introduce unrelated white-space changes into a commit. It makes it harder to review the real changes.